discovery/roles/configure_ochami/templates/cloud_init/ci-group-login_compiler_node_x86_64.yaml.j2

```diff
@@ -0,0 +1,167 @@
+        - echo "{{ cloud_init_nfs_path }}/$(hostname -s)/etc/slurm/epilog.d     etc/slurm/epilog.d      nfs defaults,_netdev 0 0" >> /etc/fstab
+        - echo "{{ cloud_init_nfs_path }}/$(hostname -s)/var/spool      /var/spool       nfs defaults,_netdev 0 0" >> /etc/fstab
+        - echo "{{ cloud_init_nfs_path }}/$(hostname -s)/etc/munge      /etc/munge       nfs defaults,_netdev 0 0" >> /etc/fstab
+        - chmod 644 /etc/fstab
+        - mount -a
+        - yes | cp /etc/slurm/epilog.d/slurmd.service /usr/lib/systemd/system/
+        - chown -R slurm:slurm /var/log/slurm
+        - chown -R slurm:slurm /var/run/slurm
+        - chown -R slurm:slurm /var/spool
+        - chown -R slurm:slurm /var/lib/slurm
+        - chown -R munge:munge /etc/munge/munge.key
+        - chmod 755 /var/log/slurm /var/run/slurm /var/spool /var/lib/slurm
+        - chmod 400 /etc/munge/munge.key
+        - chmod 755 /etc/slurm/epilog.d/
+        - chmod +x /etc/slurm/epilog.d/logout_user.sh
+        - mkdir -p /var/spool/slurmd
+        - chmod 755 /var/spool/slurmd
+        - chown -R slurm:slurm /var/spool/slurmd
+
+
+        - sed -i '/^password\s\+include\s\+password-auth/i account    required    pam_slurm_adopt.so action_no_jobs=deny' /etc/pam.d/sshd
+
+        - setenforce 0
+        - systemctl stop firewalld
+        - systemctl disable firewalld
+        - systemctl enable sshd
+        - systemctl start sshd
+        - systemctl enable munge
+        - systemctl start munge
+        - systemctl enable slurmd
+        - systemctl start slurmd
+        - systemctl daemon-reexec
+        - systemctl restart sshd
+
+        # Add NFS entry and mount
+        - mkdir -p /opt/omnia
+        - echo "{{ nfs_server_ip }}:{{ nfs_server_path }} /opt/omnia nfs defaults,_netdev 0 0" >> /etc/fstab
+        - mount -a
+
+        # UCX build and install
+        - |
+          UCX_BIN=/opt/omnia/benchmarks/ucx
+          mkdir -p /opt/omnia/compile/ucx
+          mkdir -p /opt/omnia/benchmarks/ucx
+          cd /opt/omnia/compile/ucx
+          wget --no-check-certificate https://{{ hostvars['localhost']['admin_nic_ip'] }}:2225/pulp/content/opt/omnia/offline_repo/cluster/x86_64/rhel/10.0/tarball/ucx/ucx.tar.gz
+          tar xzf ucx.tar.gz
+          cd ucx-*
+          mkdir -p build
+          cd build
+          ../contrib/configure-release --prefix=/opt/omnia/benchmarks/ucx
+          make -j 8
+          make install
+
+        # OpenMPI build and install with UCX + Slurm detection
+        - |
+          OPENMPI_INSTALL_PREFIX="/opt/omnia/benchmarks/openmpi"
+          OPENMPI_SRC="/opt/omnia/compile/openmpi"
+          mkdir -p $OPENMPI_SRC
+          mkdir -p $OPENMPI_INSTALL_PREFIX
+
+          cd $OPENMPI_SRC
+          wget --no-check-certificate https://{{ hostvars['localhost']['admin_nic_ip'] }}:2225/pulp/content/opt/omnia/offline_repo/cluster/x86_64/rhel/10.0/tarball/openmpi/openmpi.tar.gz
```

Comment:
do we support multiple openmpi tarball? are we overwriting same tar file if version  updated?